### ISSUE TITLE
Update the "Task List" styling during the installer to match the rest of Seven

### ISSUE THREAD

**Issue by quicksketch** [backdrop/backdrop-issues#2804]
## Describe your issue or idea

The task list shown in the sidebar during the installer does not match the rest of Seven, both in iconography and color scheme. We should update it to match.

---

PR https://github.com/backdrop/backdrop/pull/1969

**Comment by quicksketch** [comment]
Here's the current installer, in which the green progress bar clashes horribly with the green checkmarks on the left:

![image](https://user-images.githubusercontent.com/100206/29501346-da7352c8-85dc-11e7-9566-c101f58610ea.png)

Here is the updated design:

![image](https://user-images.githubusercontent.com/100206/29501371-fb7d900a-85dc-11e7-856f-1ddc2d2b1b1a.png)

And combined with the progress bar refresh at https://github.com/backdrop/backdrop-issues/issues/2803:

![image](https://user-images.githubusercontent.com/100206/29501392-2511ecf4-85dd-11e7-88a9-0237127c5a2b.png)

**Comment by jenlampton** [comment]
I love the new progress bar, but I don't like the green background in the sidebar. In 1.7 we removed all the light/dark green styling from reports pages, and I'd hate to see it get added back. Can we make the sidebar on the installer match the *new* reports pages instead of what was replaced? Something like this? 
<img width="1175" alt="screen shot 2017-08-24 at 11 53 09 am" src="https://user-images.githubusercontent.com/397895/29683341-dd6c1896-88c2-11e7-9a70-6d047aeaa9c7.png">
